src/main/scala/ScalaModulePlugin.scala

```diff
@@ -140,6 +143,10 @@ object ScalaModulePlugin extends AutoPlugin {
         </developer>
       </developers>
     )
+  )
+
+  lazy val scalaModuleSettingsJVM: Seq[Setting[_]] = Seq(
+    mimaPreviousVersion := None
   ) ++ mimaSettings ++ scalaModuleOsgiSettings
```

Comment:
In fact, running MiMa would have probably caught https://github.com/scala/scala-parser-combinators/issues/119 :-)